source-braintree-native/source_braintree_native/models.py

```diff
@@ -28,16 +31,16 @@ class EndpointConfig(BaseModel):
         title="Merchant ID",
     )
 
-    class ApiKey(BaseModel):
+    class ApiKey(BasicAuth):
         credentials_title: Literal["API Key"] = Field(
             default="API Key",
             json_schema_extra={"type": "string"}
         )
-        public_key: str = Field(
+        username: str = Field(
             title="Public Key",
             json_schema_extra={"secret": True},
         )
-        private_key: str = Field(
+        password: str = Field(
             title="Private Key",
             json_schema_extra={"secret": True},
         )
```

Comment:
Will changing authentication method disrupt existing captures and require an update by the user?